lib/transactionsFilter/emitBlockEventToFilterCollectionFactory.js

```diff
@@ -0,0 +1,17 @@
+const { Block } = require('@dashevo/dashcore-lib');
+
+/**
+ * @param {BloomFilterEmitterCollection} bloomFilterEmitterCollection
+ * @return {emitBlockEventOnFilterCollection}
+ */
+function emitBlockEventToFilterCollectionFactory(bloomFilterEmitterCollection) {
+  function emitBlockEventOnFilterCollection(rawBlock) {
+    const block = new Block(rawBlock);
+
```

Comment:
Why `On` and not `To`?
`emitBlockEventOnFilterCollection`
`emitBlockEventToFilterCollection`